### DISCUSSION TITLE
FR: smooth triangle/polygon/N-gon

### DISCUSSION THREAD

**Discussion by chconnor** [Bodmer/TFT_eSPI#3027] [New processor/display support or capability request] [answered]
It would be great to have a drawSmoothTriangle()/fillSmoothTriangle() function, or better, a drawSmoothPolygon()/fillSmoothPolygon(), and even better a draw/fillSmoothNGon(), where you pass in a series of points to desribe an arbitrary closed path.

(and let me know if I've missed something here... AFAICT this is not present...)

Thanks for the awesome library!

**Comment by Bodmer** [comment, marked as answer]
Yes, that is on my TODO list... along with many other things!
